.github/workflows/release.yaml

```
@@ -30,19 +45,20 @@ jobs:
             org.opencontainers.image.licenses=${{ github.event.repository.license.spdx_id }}
 
       - name: Login to GHCR
+        if: steps.semrel.outputs.version != ''
         uses: docker/login-action@v3
         with:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Build and push
-        id: docker_build_ghcr
+        if: steps.semrel.outputs.version != ''
         uses: docker/build-push-action@v5
         with:
           context: .
           file: ./Dockerfile
-          push: ${{ github.event_name != 'pull_request' }}
+          push: true
```

maybe it could run on git push without doing any actual pushing?
```suggestion
          push: ${{ steps.semrel.outputs.version != '' }}
```
We probably don't need this since building the dev image for testing covers this already?

---

the prod image also gets built in the test.yaml file